src/components/header/custom-cursors/cursor/cursor.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+
+import { ReactComponent as BlueAnimeCursor } from './assets/blueAnimeCursor.svg';
+import { ReactComponent as BlueCursor } from './assets/blueCursor.svg';
+import { ReactComponent as BlueEmptyCursor } from './assets/blueEmptyCursor.svg';
+import { ReactComponent as GrayTelegramCursor } from './assets/grayTelegramCursor.svg';
+import { ReactComponent as PurpleGradientCursor } from './assets/purpleGradientCursor.svg';
+import { ReactComponent as YellowCursor } from './assets/yellowCursor.svg';
+
+export const Cursors = {
+  BlueAnimeCursor,
+  BlueCursor,
+  BlueEmptyCursor,
+  GrayTelegramCursor,
+  PurpleGradientCursor,
+  YellowCursor,
+};
+
+export type CursorsType = keyof typeof Cursors;
+
+interface Props {
+  cursor: CursorsType;
```

Comment:
Maybe rename this prop with just `name`? I think that this prop will be clear from the context.